RegisterCommands method (#706)
jmcdo29/nest-commander: apps/docs/src/pages/en/features/commander.md

The above command is meant to be a basic example, and should not be taken as a fully fleshed out CLI example. There's error handling, input validation, and security that should all be considered. Please do not use the above command in a production environment without adding the mentioned necessities **at the very least**.

:::

## Register Commands

> Though you'll find the details are in [factory page](./factory), you must register your all commands including sub commands as providers in your module class exists somewhere in Module tree registerd to `CommandFactory`. For convenience, given that we register exapmles in Sub Commands section and set them as providers in app.module.ts that set as root module to `CommandFactory`.

Reviewer: ```suggestion
Though you'll find the implementation details in the [factory page](./factory), you must register all of your commands including the sub commands as providers in a module class that the `CommandFactory` ends up registering. For convenience, given that we register examples in Sub Commands section and set them as providers in `app.module.ts` that set as root module to `CommandFactory`.
```
